### DISCUSSION TITLE
表格的编辑框扩展功能能否关闭?

### DISCUSSION THREAD

**Discussion by BruceLuo2** [layui/layui#1555] [问答]
### 议题条件

- [x] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

疑是 BUG

### 使用版本

2.8

### 问题描述

如果一个下拉框点击小三角弹出来的这个页面进行修改, 无法正常触发相关事件
![image](https://github.com/layui/layui/assets/33817573/f123da12-e295-41f5-886e-8326ed8b2dc9)
无论是文本框还是下拉框均无法正常触发相应事件, 能否设置关闭这个展开功能
![image](https://github.com/layui/layui/assets/33817573/6b6df61d-ec71-4f15-af45-dcf08c41dad8)

### 业务代码

```auto
/
```

### 浏览器

谷歌最新

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by sentsim** [comment]
1. 尝试把 `width` 稍微设大一点
2. 把 Layui 版本升级一下：https://layui.dev/ ，早在 2.8.17 就增加了 `cellExpandedMode` 属性，默认采用的是多行展开模式。
